src/main/webapp/components/Projects/Header.js

```diff
@@ -1,8 +1,38 @@
-import React from 'react'
+import React, { useState, useEffect, useRef } from 'react'
```

Comment:
I'd keep the order of imports consistent across files. The default is usually builtin, external, and internal dependencies. 

https://dev.to/otamnitram/sorting-your-imports-correctly-in-react-213m